extensions/cornerstone/src/services/CornerstoneCacheService/CornerstoneCacheService.ts

```diff
@@ -286,7 +286,7 @@ class CornerstoneCacheService {
   }
 
   private _getCornerstoneVolumeImageIds(displaySet, dataSource): string[] {
-    const stackImageIds = this._getCornerstoneStackImageIds(displaySet, dataSource);
+    const stackImageIds = this._getCornerstoneStackImageIds(displaySet, dataSource).reverse();
```

Comment:
This is not a valid way to get the same ordering - in fact, there isn't really a good way of doing it that is equivalent to a "stack"